Userland/Services/SpiceAgent/SpiceAgent.cpp

```diff
@@ -75,13 +73,24 @@ ErrorOr<void> SpiceAgent::on_message_received()
             break;
 
         dbgln_if(SPICE_AGENT_DEBUG, "The spice server has notified us of new clipboard data of type `{}`", data_type);
+        dbgln_if(SPICE_AGENT_DEBUG, "Sending a request for data of type `{}`", data_type);
```

Comment:
Same for this new debug message, which probably should have ended up in the previous commit.